test/Conductors.Tests/RepositoryConductorTests/BulkCreateAsyncShould.cs

```diff
@@ -52,12 +44,45 @@ public async Task Throw_Argument_Null_Exception_When_Given_Null_Input()
         public async Task Throw_Argument_Exception_When_Given_Empty_Input()
         {
             // Arrange 
-            var repositoryMock = new IRepositoryMock<UserStub>();
+            var repositoryMock = new RepositoryMock<UserStub>();
             var respositoryConductor = SetupSut(repositoryMock);
 
             // Act & Assert
             await Assert.ThrowsAsync<ArgumentException>(() => respositoryConductor.BulkCreateAsync(new List<UserStub>()));
         }
 
+        [Fact]
+        public async Task Throw_OperationCanceledException_If_Canceled()
+        {
+            // Arrange 
+            var repositoryMock = new RepositoryMock<UserStub>();
+            var respositoryConductor = SetupSut(repositoryMock);
+            var cancellationTokenSource = new CancellationTokenSource();
+            var cancellationToken = cancellationTokenSource.Token;
+            cancellationTokenSource.Cancel();
+            // Act & Assert
```

Comment:
nitpick: I won't call it out everywhere, but consider adding line spaces above code comments like this for readability.

Reply:
spacing resolved